deployment/docker-compose.prod.yaml

```diff
@@ -0,0 +1,54 @@
+version: "3"
+services:
+
+  nestjs:
+    container_name: nestjs
+    build:
+      context: ../
+      dockerfile: ./deployment/Dockerfile.prod
+    depends_on:
+      - postgres
+    env_file:
+      - .env.prod
+    expose:
+      - 3000
+
+  postgres:
+    container_name: postgres
+    image: postgres:10.4
+    ports:
+      - "5432:5432"
+    volumes:
+      - db-data-prod:/var/lib/postgresql/data
+    restart: always
+    env_file:
+      - .env.prod
+
+  pgadmin:
+    container_name: pgadmin
+    image: dpage/pgadmin4:4.18
+    restart: always
+    env_file:
+      - .env.prod
+    ports:
+      - 8080:80
+    volumes:
+      - pgadmin-data-prod:/var/lib/pgadmin
+    depends_on:
+      - postgres
```

Comment:
Didn't you remove pgadmin from the prod docker-compose? 🤔

Reply:
Nop, only from the dev version! Do you want to remove it from prod too? It doesn't seem to be useful for now, so we can clean it up.

Reply:
As discussed together, I'd rather have it in the dev docker-compose than the prod one 🙂